src/main.js

```diff
@@ -108,12 +108,19 @@ function validatedSeconds() {
 };
 
 function startActivity() {
+  if (userAccomplishInput.value !== "" && userMinutesInput.value !== "" && userSecondsInput.value !== "") {
```

Comment:
With great power (bang operator) comes great responsibility (putting it in the right place)!